src/sst/core/eli/elementbuilder.h

```diff
@@ -169,14 +146,10 @@ struct CachedAllocator
     template <class... Args>
     Base* operator()(Args&&... ctorArgs)
     {
-        if ( !cached_ ) { cached_ = new T(std::forward<Args>(ctorArgs)...); }
-        return cached_;
+        static T cached(std::forward<Args>(ctorArgs)...);
+        return &cached;
```

Comment:
Another Meyers Singleton, in lieu of calling `new` if a pointer is `nullptr` (I did not see this `CachedAllocator` used anywhere, and I doubt any code calls `delete` on its returned address, so returning the address of a `static` is okay).

Reply:
If it needs to be a pointer which can be `delete`d, it simply needs to be changed to:
```c++
      static T* cached = new T(std::forward<Args>(ctorArgs)...);
      return cached;
```